hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcacheTest.java

```diff
@@ -57,11 +58,17 @@
     final static Config config = new XmlConfigBuilder().build();
 
     @BeforeClass
+    public static void setup() throws IOException {
+        System.setProperty(GroupProperty.MEMCACHE_ENABLED.getName(),"true");
```

Comment:
:+1:  as @sancar suggested.
this will also remove cleanup section in `@AfterClass`